src/category_theory/limits/fubini.lean

```diff
@@ -212,6 +212,22 @@ begin
 end
 end
 
+section
+
+variables (F) [has_limits_of_shape J C] [has_limits_of_shape K C]
+-- With only moderate effort these could be derived if needed:
+variables [has_limits_of_shape (J × K) C] [has_limits_of_shape (K × J) C]
+
+/-- The limit of `F.flip ⋙ lim` is isomorphic to the limit of `F ⋙ lim`. -/
+@[simps] noncomputable
+def limit_flip_comp_lim_iso_limit_comp_lim : limit (F.flip ⋙ lim) ≅ limit (F ⋙ lim) :=
+(limit_uncurry_iso_limit_comp_lim _).symm ≪≫
+  has_limit.iso_of_nat_iso (uncurry_obj_flip _) ≪≫
+  (has_limit.iso_of_equivalence (prod.braiding _ _) (iso.refl _)) ≪≫
+  limit_uncurry_iso_limit_comp_lim _
+
```

Comment:
It's probably worthwhile to add the following lemmas:
```lean
@[simp, reassoc]
lemma limit_flip_comp_lim_iso_limit_comp_lim_hom_π_π (j) (k) :
  (limit_flip_comp_lim_iso_limit_comp_lim F).hom ≫ limit.π _ j ≫ limit.π _ k =
  limit.π _ k ≫ limit.π _ j := sorry

@[simp, reassoc]
lemma limit_flip_comp_lim_iso_limit_comp_lim_inv_π_π (k) (j) :
  (limit_flip_comp_lim_iso_limit_comp_lim F).inv ≫ limit.π _ k ≫ limit.π _ j = 
  limit.π _ j ≫ limit.π _ k := sorry
```

Also, does `simps` actually generate useful lemmas in this case? If not, maybe we should leave it off and rely on the two suggested lemmas above.

Reply:
The `@[simps]` was working, in the sense that the application of this isomorphism in `lean-liquid` required a calculation that worked more or less `by { ext, simp }`.

However these lemmas also work, and are more idiomatic, so I'll switch to them shortly.